x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/XPackSettings.java

```diff
@@ -121,6 +121,9 @@ private XPackSettings() {
     /** Setting for enabling or disabling vectors. Defaults to true. */
     public static final Setting<Boolean> VECTORS_ENABLED = Setting.boolSetting("xpack.vectors.enabled", true, Setting.Property.NodeScope);
 
+    /** Setting for enabling or disabling geo. Defaults to true. */
+    public static final Setting<Boolean> SPATIAL_ENABLED = Setting.boolSetting("xpack.spatial.enabled", true, Setting.Property.NodeScope);
```

Comment:
we decided to remove this since we do not see a time when it is enabled or not. So the intention was that this always be enabled. Is there a reason why you think someone would want to disable this feature-set?

Reply:
Nothing aside from trying to keep the spatial module consistent with the other xpack module features (e.g., HTTP_SSL, VECTORS). Is it a good idea to force it to always be enabled or should we leave it open for the user to decide?

Reply:
I had the same thought when [I first introduced it](https://github.com/elastic/elasticsearch/pull/44389#pullrequestreview-263242829).

Ryan shared that we are actually trying to move away from this, and the first to not have an option to disable/enable is the voting_only plugin.

I, personally, do not see why anyone would want to disable it since the features are optional and do not add any extra required resources from the cluster.

Reply:
:+1:  if we're trying to remove it.